### PR TITLE
use connection string from <connectionStrings> section

### PR DESCRIPTION
**Pull request by andreymir**
#17

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@andreymir__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: src/Shared/ProviderConfiguration.cs [hunk 1]
```diff
@@ -77,6 +77,13 @@ private ProviderConfiguration(NameValueCollection config)
             // Get connection host, port and password.
             // host, port, accessKey and ssl are firest fetched from appSettings if not found there than taken from web.config
             ConnectionString = GetStringSettings(config, "connectionString", null);
+
+            if (!string.IsNullOrEmpty(ConnectionString) &&
```
Comment (SiddharthChatrolaMs) [inline src/Shared/ProviderConfiguration.cs]: Correct me if I am wrong. After this change user will provide key name for connection string as "connectionString" attribute and there will be connection string with that name in ConfigurationManager.ConnectionStrings section. Please add some unit test to make it clear.

**Comment by andreymir** [comment]
I created unit test

File: src/Shared/ProviderConfiguration.cs (hunk 1 above)
Comment (andreymir) [inline src/Shared/ProviderConfiguration.cs]: This is correct and I've added a unit test into the pull request.

**Pull request merged**